app/views/liquids/_reddit.html.erb

```diff
@@ -0,0 +1,33 @@
+<!-- app/views/liquids/_reddit.html.erb -->
+<div class="ltag__reddit--container">
+  <div class="ltag__reddit--title-container">
+    <header>
+      <div class="ltag__reddit--title">
+        <h1>
+          <img src="<%= asset_path("reddit-icon.svg") %>" alt="Reddit Logo" />
+          <a href="<%= post_url %>" target="_blank" rel="noopener noreferrer">
+            <%= title %>
+          </a>
+        </h1>
+        <div class="ltag__reddit--post-metadata">
+          <span><%= created_at %></span>
+          <span>Author: <%= author %></span>
+        </div>
+      </div>
+    </header>
+  </div>
+  <div class="ltag__reddit--body">
+    <p>
+      <% if post_hint == 'self' %>
+        <%= markdown_text %>
+      <% elsif post_hint == 'image' %>
+        <img src="<%= thumbnail %>" alt="<%= title %>">
+      <% end %>
```

Comment:
I think this might be better if it's:
```ruby
if post_hint == "image"
 <img...>
else
  <%= markdown_text %>
end
```

because I ran into an edge case where `post_hint` is `nil`. For example:
https://www.reddit.com/r/Android/comments/dvzyeb/unpopular_opinion_3button_navigation_is_just_flat.json